src/screens/admin/FieldMappingConfig.tsx

```
@@ -0,0 +1,1046 @@
+		const newFieldMappings = [...fieldMappings];
+		newFieldMappings[fieldIndex].fieldId = fieldId;
+		newFieldMappings[fieldIndex].isExpanded = !!fieldId;
+		setFieldMappings(newFieldMappings);
+
+		// Clear related errors
+		const newErrors = { ...errors };
+		delete newErrors[`field_${fieldIndex}`];
+		setErrors(newErrors);
+	};
+
+	const toggleFieldExpansion = (fieldIndex) => {
```

ESLint

[error] 'toggleFieldExpansion' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)